### ISSUE TITLE
Android Bootcamp Project Todo - Ready for Review

### ISSUE THREAD

**Issue by nbharathram** [nbharathram/SampleTodo#1]
My Simple To Do application is complete. Please review and provide me your feedback, /cc @codepathreview @codepath

**Comment by nesquena** [comment]
Hi Bharath,

Thanks for submitting the Todo app. Can you transfer your README to markdown. You can review the section in the prework about [the readme](http://courses.codepath.com/snippets/intro_to_android/prework#heading-4-submitting-the-project) for your project. In particular, check out this [README template](http://courses.codepath.com/snippets/intro_to_android/readme_templates/prework_readme.md?raw=true) and be sure to [include a GIF recorded with LiceCap](http://www.cockos.com/licecap/). Please ping me back here once that's complete.

**Comment by nbharathram** [comment]
Sure Nathan. After submitting the current work, I have gone through few
other Readme files in markdown and thought of modifying mine as well. Can I
do this by tomorrow evening?

Thank you very much once again for your time and guidance.

Regards
Bharath Narasingu

**Comment by nesquena** [comment]
Yes, that works for us, please ping me back here once it's ready for review.

**Comment by nbharathram** [comment]
Hi Nathan,

Sorry couldn't complete the Readme file today because of an emergency.

Will definitely complete this tomorrow and submit the same for review.

Request you for the additional time.

Thanks
Bharath

**Comment by nbharathram** [comment]
Hi Nathan,

Added the README.MD file as per the instructions along with the recorded
GIF file. Please review and let me know your comments. This exercise is
already turning out to be a good education for me.

Here is the link to the Readme.md file.

https://github.com/nbharathram/SampleTodo/blob/master/README.md

Thank you very much for your time.

Regards,
Bharath Narasingu

**Comment by nesquena** [comment]
Bharath,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer  learn the practical elements of Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and to **schedule a short 10 minute phone conversation** with us. Navigate to [today's date and onward from there](https://calendar.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose a 15-minute slot. Let us know if none of those times work. 

With regards to extensions to implement, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app.

Once you select a slot, can you make sure to **include best number** to reach you at to the invite? Look forward to chatting soon!
